LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016
```

Comment:
2017?  I think we should update scaffold.

Reply:
It's a time travelling role! I will update here and the scaffold.